### DISCUSSION TITLE
API to get information about a non-core cask

### DISCUSSION THREAD

**Discussion by koterpillar** [Homebrew/discussions#3618] [Casks]
How do I get the latest version of a cask like `homebrew/cask-fonts/font-fira-code` via the JSON API?

I know I can use `https://formulae.brew.sh/api/formula/XXXX.json` and `https://formulae.brew.sh/api/cask/XXXX.json`, but can't get those to work with the font packages.

**Comment by Rylan12** [comment]
At the moment, the only cask tap that's available via the API is `homebrew/homebrew-cask`, so this isn't currently possible. We are planning on adding the remaining cask taps at some point in the future, though
